[BUG] NLTK does not support Turkish

### Description

```
PAPERLESS_OCR_LANGUAGE=tur+deu+eng
PAPERLESS_OCR_LANGUAGES=eng deu tur
```
It was working without errors when `PAPERLESS_OCR_LANGUAGE=eng+deu+tur` was set but process of accentric Turkish characters were incorrect such as YÜKSEKÖĞRETİM were recognized as YÜKSEKÖÐRETÝM, probably Turkish recognized as English. 
However then when I switch to `PAPERLESS_OCR_LANGUAGE=tur+deu+eng` order it gives the `language 'turkish' is not supported error`. 

If related as I see Turkish has no NLTK support. [nltk.stem.SnowballStemmer](https://www.nltk.org/api/nltk.stem.SnowballStemmer.html?highlight=stopwords#nltk.stem.SnowballStemmer)


### Steps to reproduce

Set env
 ```
PAPERLESS_OCR_LANGUAGE=tur+deu+eng
PAPERLESS_OCR_LANGUAGES=eng deu tur
```

### Webserver logs

```bash
[2024-07-14 23:41:27,073] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-z2i7jo3c

[2024-07-14 23:41:27,074] [ERROR] [paperless.tasks] ConsumeTaskPlugin failed: CleanShot 2024-07-14.png: The following error occurred while storing document CleanShot 2024-07-14.png after parsing: The language 'turkish' is not supported.

Traceback (most recent call last):

  File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 327, in main_wrap

    raise exc_info[1]

  File "/usr/src/paperless/src/documents/consumer.py", line 651, in run

    document_consumption_finished.send(

  File "/usr/local/lib/python3.11/site-packages/django/dispatch/dispatcher.py", line 176, in send

    return [

           ^

  File "/usr/local/lib/python3.11/site-packages/django/dispatch/dispatcher.py", line 177, in <listcomp>

    (receiver, receiver(signal=self, sender=sender, **named))

               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/src/paperless/src/documents/signals/handlers.py", line 95, in set_correspondent

    potential_correspondents = matching.match_correspondents(document, classifier)

                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/src/paperless/src/documents/matching.py", line 37, in match_correspondents

    pred_id = classifier.predict_correspondent(document.content) if classifier else None

              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/src/paperless/src/documents/classifier.py", line 413, in predict_correspondent

    X = self.data_vectorizer.transform([self.preprocess_content(content)])

                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/src/paperless/src/documents/classifier.py", line 380, in preprocess_content

    self._stemmer = SnowballStemmer(settings.NLTK_LANGUAGE)

                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/local/lib/python3.11/site-packages/nltk/stem/snowball.py", line 106, in __init__

    raise ValueError(f"The language '{language}' is not supported.")

ValueError: The language 'turkish' is not supported.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):

  File "/usr/src/paperless/src/documents/tasks.py", line 151, in consume_file

    msg = plugin.run()

          ^^^^^^^^^^^^

  File "/usr/src/paperless/src/documents/consumer.py", line 714, in run

    self._fail(

  File "/usr/src/paperless/src/documents/consumer.py", line 302, in _fail

    raise ConsumerError(f"{self.filename}: {log_message or message}") from exception

documents.consumer.ConsumerError: CleanShot 2024-07-14.png: The following error occurred while storing document CleanShot 2024-07-14.png after parsing: The language 'turkish' is not supported.
```


### Browser logs

_No response_

### Paperless-ngx version

2.11.0

### Host OS

Linux-5.2.0-x86_64-with-glibc2.36

### Installation method

Docker - official image

### System status

```json
{
    "pngx_version": "2.11.0",
    "server_os": "Linux-5.2.0-x86_64-with-glibc2.36",
    "install_type": "docker",
    "storage": {
        "total": 126690590720,
        "available": 76533125120
    },
    "database": {
        "type": "postgresql",
        "url": "paperless",
        "status": "OK",
        "error": null,
        "migration_status": {
            "latest_migration": "paperless_mail.0025_alter_mailaccount_owner_alter_mailrule_owner_and_more",
            "unapplied_migrations": []
        }
    },
    "tasks": {
        "redis_url": "redis://broker:6379",
        "redis_status": "OK",
        "redis_error": null,
        "celery_status": "OK",
        "index_status": "OK",
        "index_last_modified": "2024-07-14T23:30:10.802913+02:00",
        "index_error": null,
        "classifier_status": "OK",
        "classifier_last_trained": "2024-07-14T21:05:27.790758Z",
        "classifier_error": null
    }
}
```


### Browser

Chrome

### Configuration changes

_No response_

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

--- a/src/paperless/settings.py
+++ b/src/paperless/settings.py
@@ -1126,6 +1126,10 @@ def _get_nltk_language_setting(ocr_lang: str) -> Optional[str]:
 
     NLTK Languages:
       - https://www.nltk.org/api/nltk.stem.snowball.html#nltk.stem.snowball.SnowballStemmer
+      - https://raw.githubusercontent.com/nltk/nltk_data/gh-pages/packages/tokenizers/punkt.zip
+      - https://raw.githubusercontent.com/nltk/nltk_data/gh-pages/packages/corpora/stopwords.zip
+
+    The common intersection between all languages in those 3 is handled here
 
     """
     ocr_lang = ocr_lang.split("+")[0]
@@ -1142,7 +1146,6 @@ def _get_nltk_language_setting(ocr_lang: str) -> Optional[str]:
         "rus": "russian",
         "spa": "spanish",
         "swe": "swedish",
-        "tur": "turkish",
     }
 
     return iso_code_to_nltk.get(ocr_lang)

